driver/executor/executor.go

```diff
@@ -357,9 +443,6 @@ func scheduleNodeEvents(node *parser.Node, queue *eventQueue, net driver.Network
 					if instance == nil {
 						return nil
 					}
-					if err := net.KillNode(*instance); err != nil {
```

Comment:
I guess there is not really a test scenario in the release tests that used the "kill" feature before, as we saw that it was buggy. We should be aware that this PR changes the interpretation of "kill" from a sudden stop to a graceful shutdown, informing all peers that the node is no longer present [here](https://github.com/0xsoniclabs/norma/blob/b833757b87cd15ca1cc7b062cf95ef0c2095029b/driver/network/local/local.go#L244-L267), before stopping the node.

If the assumption that "kill" was not used before is true, this redefinition is fine. If it is used, we should check whether the old definition of a sudden, unannounced stop needs to be preserved.